src/components/SpecialResolution/HelpSpecialResolution.vue

```diff
@@ -0,0 +1,200 @@
+            <ul>
+              <li>
+                The name of the Cooperative Association, incorporation number, date the resolution was passed,
+                and the full text of the resolution which clearly indicates the changes being made.
+              </li>
+              <li>
+                The signature of a current director, officer or lawyer of the Cooperative Association and
+                the date of the signature.
+              </li>
+            </ul>
+          </div>
+          <br>
+          <br>
+          <!-- Special Resolution Example Form -->
+          <header id="sample-resolution-header">
+            <h2>{{ getSpecialResolutionResource.header }}</h2>
+          </header>
+          <p
+            class="section-description mt-2"
+            v-html="getSpecialResolutionResource.text"
+          />
+          <div class="mt-4">
+            <v-card
+              flat
+              class="py-8 px-6"
+            >
+              <div class="d-flex flex-column flex-sm-row justify-center align-center">
+                <img
+                  src="@/assets/images/BCRegistries_CoopSpecialResolution-x2.png"
+                  :alt="getSpecialResolutionResource.label"
+                  class="preview-image"
+                >
+                <div class="px-8" />
+                <div class="download-link-container py-5">
+                  <v-icon
+                    color="primary"
+                    class="mt-n1"
+                  >
+                    mdi-file-pdf-outline
+                  </v-icon>
+                  <a
+                    :href="documentURL"
+                    download
+                    class="ml-1"
+                  >{{ getSpecialResolutionResource.label }}</a>
+                </div>
+              </div>
+            </v-card>
+          </div>
+          <div
+            class="help-btn bottom"
+            @click="helpToggle = !helpToggle"
+          >
+            Hide Help
+          </div>
+        </section>
+      </v-expand-transition>
+    </div>
+    <v-divider
+      v-if="!helpToggle"
+      class="mx-4 mt-8"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Getter } from 'pinia-class'
+import { Component } from 'vue-property-decorator'
+import { ResourceIF, SpecialResolutionSampleFormIF } from '@/interfaces/'
+import { useStore } from '@/store/store'
+
+@Component({
+  components: {
+  }
+})
+export default class HelpSpecialResolution extends Vue {
+  @Getter(useStore) getResource!: ResourceIF
+  helpToggle = false
+
+  get getSpecialResolutionResource (): SpecialResolutionSampleFormIF {
+    return this.getResource.changeData?.specialResolution?.sampleFormSection || {}
+  }
+
+  /** download URL for pdf file */
+  get documentURL (): string {
+    /**
+     * In session is stored the BASE_URL with business ID
+     * So we are taking from process.env.BASE_URL
+     */
+    return process.env.BASE_URL +
```

Comment:
Not sure if this will work deployed

Reply:
Try this instead:
https://github.com/bcgov/business-create-ui/blob/f3cf108f26e5a9ddf932c107e5ec155fa547520d/src/components/Dissolution/CompleteAffidavit.vue#L224

Reply:
Fixed, thanks.